.github/stale.yml

```diff
@@ -1,24 +1,42 @@
 # https://github.com/marketplace/stale
+# https://github.com/probot/stale
+
+issues:
+    # Number of days of inactivity before an issue becomes stale
+    daysUntilStale: 30
+    # Number of days of inactivity before a stale issue is closed
+    daysUntilClose: 7
+    # Issues with these labels will never be considered stale
+    exemptLabels:
+      - Important
+      - Priority
+    # Comment to post when marking an issue as stale. Set to `false` to disable
+    markComment: >
+This issue has been automatically marked as stale because it hasn't had any recent activity.
+This issue will be closed in 7 days if no further activity occurs.
+Thank you for your contributions, Pytorch Lightning Team!
```

Comment:
```suggestion
      This issue has been automatically marked as stale because it hasn't had any recent activity.
      This issue will be closed in 7 days if no further activity occurs.
      Thank you for your contributions, Pytorch Lightning Team!
```